internal/profiler/profiler.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 // Init starts the supported profilers IFF the environment variable is set.
-func Init(svcName, version string, blockProfileRate int) {
+func Init(svcName, version string) {
```

Comment:
are we avoiding importing `zoekt` here? Can we just remove version from here and just always set it to `zoekt.Version`? Same for InitLightweight